src/rp2_common/pico_sleep/include/pico/sleep.h

```diff
@@ -50,6 +53,12 @@ static inline void sleep_run_from_xosc(void) {
     sleep_run_from_dormant_source(DORMANT_SOURCE_XOSC);
 }
 
+#if !PICO_RP2040
+static inline void sleep_run_from_lpsc(void) {
```

Comment:
Is the `_lpsc` here supposed to be `_lposc` ?

Reply:
yes